package/yast2-update.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Aug 22 10:59:42 UTC 2017 - knut.anderssen@suse.com
+
+- Show also legacy root filesystems in the selectable list for
```

Comment:
I would say something like "Show legacy root filesystems in the list of systems to upgrade from".